src/CoreBluetooth/GuidWrapper.cs

```diff
@@ -30,28 +30,13 @@ public void ConnectPeripheral (CBPeripheral peripheral, PeripheralConnectionOpti
 		}
 
 #if !TVOS && !WATCH
-		[Deprecated (PlatformName.iOS, 7, 0)]
-		[Obsoleted (PlatformName.iOS, 9, 0, message : "Use 'RetrievePeripheralsWithIdentifiers' instead.")]
+		[Obsolete ("Always throws NotSupportedException (). (not a public API).")]
```

Comment:
Obsolete code should be under `#if !XAMCORE_4_0` so we don't carry those over when a breaking change happens